pkg/helpers/helpers.go

```diff
@@ -189,9 +189,9 @@ func IsBpfAppStateConditionFailure(conditions *[]metav1.Condition) bool {
 		log.Info("more than one condition found", "numConditions", numConditions)
 	}
 
-	if (*conditions)[0].Type == string(bpfmaniov1alpha1.BpfAppCondSuccess) {
-		return false
-	} else {
+	if (*conditions)[0].Type == string(bpfmaniov1alpha1.BpfAppCondPending) {
 		return true
+	} else {
```

Comment:
no need for else here and the same for failed

Reply:
Done